apps/src/dance/ai/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const DANCE_AI_SOUNDS = [
```

Comment:
I split this into a separate constants file in order to use the `as const` feature of typescript - logged a tech debt item to go back and consolidate this with the existing `dance/constants` file.